### PR TITLE
Add changelog entry for patch "netbox: add "path" to index schema"

### PR DESCRIPTION
**Pull request by olegrok**
Follow-up #5451

### CODE REVIEW

File: changelogs/unreleased/fix-netbox-index-schema-path.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## bugfix/core
+
+* Fixed missing "path" value of index schema fetched by netbox.
```
Comment (Totktonada) [inline changelogs/unreleased/fix-netbox-index-schema-path.md:3]: Issue number, please (in the form `(gh-xxxx)`).

File: changelogs/unreleased/fix-netbox-index-schema-path.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## bugfix/core
```
Comment (Totktonada) [inline changelogs/unreleased/fix-netbox-index-schema-path.md:1]: Re commit message: I hope this change may be described in 50 symbols.

File: changelogs/unreleased/fix-netbox-index-schema-path.md (hunk 1 above)
Comment (olegrok) [reply changelogs/unreleased/fix-netbox-index-schema-path.md]: Fixed

File: changelogs/unreleased/fix-netbox-index-schema-path.md (hunk 2 above)
Comment (olegrok) [reply changelogs/unreleased/fix-netbox-index-schema-path.md:1]: I did the same as in https://github.com/tarantool/tarantool/commit/84a2f70794bc0672599109f50a8caa66436b643d

Updated.

Comment (Totktonada) [reply changelogs/unreleased/fix-netbox-index-schema-path.md:1]: Personally I dislike issue / commit numbers in commit titles (a subject should be descriptive), but I don't want to bother you with that.

Comment (Totktonada) [inline changelogs/unreleased/fix-netbox-index-schema-path.md:1]: Or core, or lua. Don't know.

**Review by Totktonada — APPROVED** [review]

File: changelogs/unreleased/fix-netbox-index-schema-path.md (hunk 2 above)
Comment (Korablev77) [reply changelogs/unreleased/fix-netbox-index-schema-path.md:1]: Core is ok.

**Review by Korablev77 — APPROVED** [review]

File: changelogs/unreleased/fix-netbox-index-schema-path.md (hunk 2 above)
Comment (Totktonada) [reply changelogs/unreleased/fix-netbox-index-schema-path.md:1]: Okay.

Comment (olegrok) [reply changelogs/unreleased/fix-netbox-index-schema-path.md:1]: The same as for another netbox issue (https://github.com/tarantool/tarantool/blob/master/changelogs/unreleased/fix-netbox-memory-corruption.md)

**Comment by Korablev77** [comment]
Pushed to master, 2.7 and 2.6

**Pull request closed without merging**